packages/migrate/src/commands/DbPull.ts

```diff
@@ -204,8 +206,9 @@ Set composite types introspection depth to 2 levels
       )
       .run()
 
-    // Re-Introspection is not supported on MongoDB
+    const schemasFromCliArgs = args['--schemas']?.split(',')
```

Comment:
Reason why to do this out here instead of before the `introspect()` call?

Reply:
None now, there was one before though (code was checking it for check if the preview feature was also set before, but this is now done in engines)

Reply:
I removed it